app/src/main/res/values/dimens.xml

```diff
@@ -1,2 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources></resources>
\ No newline at end of file
+<resources>
+    <dimens name="dmn_wid_hig">125dp</dimens>
+    <dimens name="dmn_margin_top">110dp</dimens>
+    <dimens name="dmn_padding">0dp</dimens>
+    <dimens name="dmn_margin_tp">75dp</dimens>
+    <dimens name="dmn_margin_lr">20dp</dimens>
+    <dimens name="dmn_margin_tpe">40dp</dimens>
```

Comment:
'dimens' is not a valid tag type. Replace every 'dimens' with `dimen`. Checks should pass then.